scene/register_scene_types.cpp

```diff
@@ -390,6 +391,8 @@ void register_scene_types() {
 	ClassDB::register_class<RigidBody>();
 	ClassDB::register_class<KinematicCollision>();
 	ClassDB::register_class<KinematicBody>();
+	ClassDB::register_class<SpringArm>();
```

Comment:
I think you should probably add a gizmo to this, where you can edit the length (and optionally the margin if you use that)

Reply:
Can you point me to an example of where this is done? I have no idea on how to implement that. Probably also  @groud  can help me?

Reply:
check the ray shape gizmo, I think that you can copy most part of code from there

Reply:
@AndreaCatania I'm not sure the ray shape gizmo is that good, I opened an issue on that [here](https://github.com/godotengine/godot/issues/19793)

Reply:
It can be improved, but the basic code is almost the same